AirCasting/Utils/Concurrency.swift

```diff
@@ -0,0 +1,30 @@
+// Created by Lunar on 9.06.25.
+//
+
+import Foundation
+
+func waitFor<T>(_ operation: @escaping () async throws -> T) throws -> T {
```

Comment:
There has to be a library for that... right? RIGHT?! 😂

Reply:
AFAIK as this is not a preferred practice (but quite used in legacy projects :D), you don't typically have libraries offering these things, but if there is a Swift one that does, I will change it (checked it quickly but didn't find any). In the Android project we had to add a similar thing when we were in the process of refactoring everything to use suspending functions, and later removed this when we were done